bin/test-cleanup

```diff
@@ -9,7 +11,6 @@ fi
 
 echo "cleaning up namespace [${linkerd_namespace}] and associated test namespaces"
 
-kubectl delete ns "$linkerd_namespace"
-for ns in $(kubectl get ns | grep "^$linkerd_namespace-" | cut -f1 -d ' '); do
-  kubectl delete ns "$ns"
-done
+# `kubectl delete ns` will fail if there are no matching matching namespaces,
+# causing this script to exit with an error status.
+kubectl get ns | cut -f1 -d ' ' | grep -E "^$linkerd_namespace(-|$)" | xargs kubectl delete ns
```

Comment:
Thanks for updating! This still doesn't return an error for me if there are no matching namespaces:

```
$ ./bin/test-cleanup asdf; echo $?
cleaning up namespace [asdf] and associated test namespaces
0
```

I think the issue is that my version of kubectl behaves like this:

```
$ echo "" | xargs kubectl delete ns; echo $?
0
```